clusters/app.ci/supplemental-ci-images/telco-bastion-cs9.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-bastion-cs9
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    namespace: ci
+    name: telco-bastion-cs9
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: telco-bastion-cs9:latest
+    source:
+      dockerfile: |
+        FROM registry.ci.openshift.org/openshift/centos:stream9
+        ENV GOCACHE=/go/.cache \
+            GOARM=5 \
+            GOPATH=/go \
+            LOGNAME=deadbeef
+        ENV GOROOT=/usr/local/1.20/go
+        ENV PATH=$PATH:$GOROOT/bin:$GOPATH/bin
+        ENV HOME /output
+        RUN dnf install -y ansible-core wget make jq which && \
+            dnf clean all && \
+            wget https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.11.9/openshift-client-linux.tar.gz && \
+            tar -zxvf openshift-client-linux.tar.gz && \
+            mv oc /usr/local/bin/ && \
+            mv kubectl /usr/local/bin/ && \
+            rm -f openshift-client-linux.tar.gz && \
+            mkdir /output && \
+            chmod -R g+rwx /output && \
+            chmod g+w /etc/passwd && \
+            echo 'echo default:x:$(id -u):$(id -g):Default Application User:/output:/sbin/nologin\ >> /etc/passwd' > /output/fix_uid.sh && \
+            chmod g+rwx /output/fix_uid.sh
+        RUN mkdir -p /usr/local/{1.19,1.20,1.21.6} && \
+            for version in 1.19 1.20 1.21.6; do \
+            wget https://golang.org/dl/go${version}.linux-amd64.tar.gz && \
+            tar -C /usr/local/${version} -xzf go${version}.linux-amd64.tar.gz && \
+            rm go${version}.linux-amd64.tar.gz; \
+            done
+        RUN mkdir -p $GOPATH/bin && \
+            chmod g+xw -R $GOPATH && \
+            chmod g+xw -R $(go env GOROOT)
+        ENV GOFLAGS='-mod=vendor'
+      type: Dockerfile
```

Comment:
I'd suggest to prepare env files that contain all required variable set for specific golang version, so we can just "source" them in the job. For example we do in job:
```bash
source /etc/golang-1.21
```
and in file `/etc/golang-1.21`:
```bash
#!/bin/sh
export GOROOT=...
export ...
```

Reply:
Done

~~~
bash-5.1# go version
bash: go: command not found


bash-5.1# cat /output/golang-1.21.6
export GOLANG_VERSION=1.21.6
export GOROOT=/usr/local/${GOLANG_VERSION}/go
export PATH=$PATH:/usr/local/${GOLANG_VERSION}/go/bin:/go/bin

bash-5.1# . /output/golang-1.21.6 

bash-5.1# go version
go version go1.21.6 linux/amd64
~~~